design/ceph/object/store.md

```diff
@@ -73,7 +74,149 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
+    application: rgw
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+ metadata:
+    name: rgw-data-pool
+  spec:
+    failureDomain: osd
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 2
+    application: rgw
+```
+
+The `.rgw.root` pool must be defined before the object store is created. This pool is used for storing the root info for all object stores.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-root
+  spec:
+    name: .rgw.root
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+```
+
+The pools for this configuration will be created as below:
+
+```bash
+# ceph osd pool ls
+.rgw.root
+rgw-meta-pool
+rgw-data-pool
+```
+
+And the pool configuration in zone is as below:
+
+```json
+# radosgw-admin zone get --rgw-zone=my-store
+{
+    "id": "2220eb5f-2751-4a51-9c7d-da4ce1b0e4e1",
+    "name": "my-store",
+    "domain_root": "rgw-meta-pool:my-store.meta.root",
+    "control_pool": "rgw-meta-pool:my-store.control",
+    "gc_pool": "rgw-meta-pool:my-store.log.gc",
+    "lc_pool": "rgw-meta-pool:my-store.log.lc",
+    "log_pool": "rgw-meta-pool:my-store.log",
+    "intent_log_pool": "rgw-meta-pool:my-store.log.intent",
+    "usage_log_pool": "rgw-meta-pool:my-store.log.usage",
+    "roles_pool": "rgw-meta-pool:my-store.meta.roles",
+    "reshard_pool": "rgw-meta-pool:my-store.log.reshard",
+    "user_keys_pool": "rgw-meta-pool:my-store.meta.users.keys",
+    "user_email_pool": "rgw-meta-pool:my-store.meta.users.email",
+    "user_swift_pool": "rgw-meta-pool:my-store.meta.users.swift",
+    "user_uid_pool": "rgw-meta-pool:my-store.meta.users.uid",
+    "otp_pool": "rgw-meta-pool:my-store.otp",
+    "system_key": {
+        "access_key": "",
+        "secret_key": ""
+    },
+    "placement_pools": [
+        {
+            "key": "default-placement",
+            "val": {
+                "index_pool": "rgw-metadata-pool:my-store.buckets.index",
+                "storage_classes": {
+                    "STANDARD": {
+                        "data_pool": "rgw-data-pool:my-store.buckets.data"
+                    }
+                },
+                "data_extra_pool": "rgw-data-pool:my-store.buckets.non-ec", #only pool is not erasure coded, otherwise use different pool
+                "index_type": 0,
+                "inline_data": "true"
+            }
+        }
+    ],
+    "realm_id": "65a7bf34-42d3-4344-ac40-035e160d7f9e",
+    "notif_pool": "rgw-meta-pool:my-store.notif"
+}
+```
+
+Manual steps involved to add this feature assuming zone and pool are already created:
+
+```bash
+
+# radosgw-admin zone get --rgw-zone=my-store > zone.json
+
+# modify the zone.json to add the pools as above
+
+# radosgw-admin zone set --rgw-zone=my-store --infile=zone.json
+
+# radosgw-admin period update --commit
+```
+
+After deleting the object store the data in the rados namespace can deleted by following commands:
```

Comment:
The operator would automate these during uninstall, right? Please review [this comment](https://github.com/rook/rook/pull/12140#issuecomment-1828777007) for the suggested setting related to  uninstall.